DESCRIPTION

```diff
@@ -1,11 +1,22 @@
 Package: ClimDown
-Version: 0.0.1
-Date: 2016-??-??
+Version: 1.0.8
+Date: 2021-06-14
 Title: Climate Downscaling Library for Daily Climate Model Output
-Authors@R: c(person("Alex Cannon", role = "aut", email = "acannon@uvic.ca"),
-	     person("James Hiebert", role = c("aut", "cre"), email = "hiebert@uvic.ca"),
-	     person("Arelia Werner", role = "aut", email = "wernera@uvic.ca"),
-	     person("Stephen Sobie", role = "aut", email = "ssobie@uvic.ca"))
+Authors@R: c(person("Nic Annau", role = "aut", email = "nannau@uvic.ca"),
+	     person("Alex Cannon", role = "aut", email = "acannon@uvic.ca"),
+	     person("James Hiebert", role = "aut", email = "hiebert@uvic.ca"),
+             person("Cairo Sanders", role = "ctb", email = "cairosanders@uvic.ca"),
+	     person("Arelia Schoeneberg", role = "aut", email = "wernera@uvic.ca"),
+             person("Stephen Sobie", role = "aut", email = "ssobie@uvic.ca"),
+             person("Lee Zeman", role = c("aut","cre"), email = "lzeman@uvic.ca"))
```

Comment:
Looks like some of the formatting here is a bit off, will this have any kind of impact on the build?